### DISCUSSION TITLE
Caching of ISR compared to pre-render on Vercel

### DISCUSSION THREAD

**Discussion by leoj3n** [sveltejs/kit#11918] [Q&A]
I'm just curious if there are any implementation detail differences when comparing ISR to **pre-rendering**, specifically in regards to caching. These docs make it sound like they are handled pretty much the same:

- https://kit.svelte.dev/docs/adapter-vercel#incremental-static-regeneration
  - (ISR) provides the performance and cost advantages of prerendered content.
- https://vercel.com/docs/frameworks/sveltekit#incremental-static-regeneration-isr
  - ISR generated pages are both cached and persisted to durable storage.
- https://vercel.com/docs/incremental-static-regeneration
  - ISR allows Vercel to cache generated pages in every region on our global Edge Network and persist files into durable storage.
- https://vercel.com/docs/incremental-static-regeneration#differences-between-isr-and-cache-control-headers
  - ISR has cache shielding built-in automatically, which helps improve the cache HIT ratio. The cache for your ISR route's Vercel Function output is distributed globally. In the case of a cache MISS, it looks up the value in a single, global bucket. With only cache-control headers, caches expire (by design) and are not shared across regions.

However, it's not totally clear if they are somehow cached differently, or if they would both be on "durable storage" and served the same. If anyone has any insight I'm just curious if there are any advantages, or at least differences, to ISR with a long timeout compared to pre-render when it comes to caching on Vercel.

**Comment by madeleineostoja** [comment]
Also came here to ask the same thing, because I'm noticing a _lot_ more cache MISS using ISR compared to standard pre-rendering
